.github/workflows/build.yml

```diff
@@ -123,7 +123,7 @@ jobs:
     strategy:
       fail-fast: true
       matrix:
-        goos: [freebsd, windows, linux, darwin, openbsd]
+        goos: [freebsd, windows, linux, darwin]
```

Comment:
do we not need `openbsd` anymore? I remember us adding this a while back: https://github.com/hashicorp/terraform-provider-aws/issues/19138